dhis-2/pom.xml

```diff
@@ -407,6 +407,9 @@
 
         <profile>
             <id>sonarqube</id>
+            <properties>
+                <skipTests>true</skipTests>
```

Comment:
If Sonarqube needs jacoco coverage report, that would prevent it from getting them.

Reply:
it does for coverage, but the coverage has been broken for a while now. @stian-sandvold made an attempt to fix that, but I believe it was reverted. So I think it's fine to skip tests for now, until the coverage is fixed.

Reply:
As soon as the tests are not skipped anymore, the analyze will run and everything will still work fine